packages/node-core/src/indexer/blockDispatcher/block-dispatcher.ts

```diff
@@ -86,27 +86,33 @@ export abstract class BlockDispatcher<B, DS>
     this.isShutdown = true;
     this.processQueue.abort();
   }
-  enqueueFatBlocks(fatBlocks: B[], start: number, end: number): void | Promise<void> {
-    logger.info(`Enqueueing fat blocks ${start}...${end}, total ${fatBlocks.length} blocks`);
-    // Still update this number, so loop can continue
-    this.latestBufferedHeight = end;
-    this.fatBlocksQueue.putMany(fatBlocks);
-    void this.processFatBlocksFromQueue();
-  }
 
-  enqueueBlocks(heights: number[], latestBufferHeight?: number): void {
+  enqueueBlocks(heightsBlocks: (B | number)[], latestBufferHeight?: number): void {
```

Comment:
```suggestion
  enqueueBlocks(heightsBlocks: B[] | number[], latestBufferHeight?: number): void {
```

Alternatively change behaviour to support an array of both types